src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,67 @@
+import { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+import { PeopleTable } from '../../components/PeopleTable';
+import { ErrorType } from '../../types/ErrorType';
+
+const complementPeopleList = (people: Person[]) => {
+  return people.map((person) => (
+    {
+      ...person,
+      father: people.find(father => person.fatherName === father.name),
+      mother: people.find(mother => person.motherName === mother.name),
+    }
+  ));
+};
+
+export const PeoplePage = () => {
+  const { slug = '' } = useParams();
+
+  const [peopleList, setPeopleList] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState(false);
+  const [errorType, setErrorType] = useState<ErrorType>(ErrorType.none);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getPeople()
+      .then(people => {
```

Comment:
Let’s use `async await` all over the code. It gives you a simpler way to work with asynchronous promise-based code